### DISCUSSION TITLE
Can we have exhaustScan operator?

### DISCUSSION THREAD

**Discussion by kamilzielinskidev** [ReactiveX/rxjs#6667] [Ideas / Feature request]
The operator would be helpfull for throttling for example request until the previous one is finished.
There is some external implementation from this guy: https://github.com/svenvanheugten/rxjs-exhaustscan yet it is not included in rxjs officially

**Comment by kwonoj** [comment]
This request is moved to discussions since gh issue is rxjs core bug tracking only.

**Reply by kamilzielinskidev** [reply]
ok sorry, didn't know about it, thanks for moving it
